### ISSUE TITLE
Data repositories

### ISSUE THREAD

**Issue by luiztauffer** [luiztauffer/alyx#6]
map the files from possibly 3 source repositories: 

- Google drive
- Globus Connect (already present, see DataRepository table under Data app)
- Local file system

**Comment by bendichter** [comment]
If the google drive is mounted this shouldn't be very different from a local filesystem, right? In both cases, we'd need a base location and a formula for the data path within that location.

**Comment by bendichter** [comment]
I think it makes sense to be in line with DANDI, which is trying to impose the structure `'sub-{}/sub-{}_ses-{}.nwb'.format(subject_id, subject_id, session_id)` for all the data files. See https://gui.dandiarchive.org/#/file-browser/folder/5ea1a111a8c7364fa9308f4f

**Comment by luiztauffer** [comment]
I agree with the Google drive mounted locally, but that doesn't need to be the case, right? If they have the infinite storage as you mentioned, the'd probably have to access it through an API.
I'll take a look at the DANDI link later
